test/apiv2/python/rest_api/fixtures/podman.py

```diff
@@ -46,16 +46,25 @@ def __init__(self):
         buf = json.loads(
             """
             {
-              "cniVersion": "0.3.0",
+              "cniVersion": "1.0.0",
               "name": "podman",
               "plugins": [{
                   "type": "bridge",
                   "bridge": "cni0",
                   "isGateway": true,
                   "ipMasq": true,
                   "ipam": {
-                    "type": "host-local",
-                    "subnet": "10.88.0.0/16",
+                    "type": "static",
+                    "addresses": [
+                        {
+                            "address": "10.88.0.0/16",
+                            "gateway": "10.88.0.1"
+                        },
+                        {
+                            "address": "10.89.0.0/16",
+                            "gateway": "10.88.0.1"
+                        }
+                    ],
```

Comment:
Do not modify the default network here, likely cni is not even used here since we prefer netavark as of podman v4.0.

I recommend to create a test case which creates a network with two ipv4 subnets and then run a container on it. It is not safe to assume that cni will be used.

Reply:
Done.